package.json

```diff
@@ -68,7 +68,10 @@
   },
   "dependencies": {
     "@emotion/core": "10.0.22",
+    "@emotion/css": "10.0.22",
+    "@emotion/is-prop-valid": "0.8.5",
     "@emotion/styled": "10.0.23",
+    "@emotion/styled-base": "10.0.24",
```

Comment:
Can you explain why do we need all those deps?

Reply:
gonna move them to the ignored externals instead